Makefile

```diff
@@ -291,13 +291,8 @@ clean-docker: ## Deletes the docker containers for local development.
 govet: ## Runs govet against all packages.
 	@echo Running GOVET
 	$(GO) get -u golang.org/x/tools/go/analysis/passes/shadow/cmd/shadow
```

Comment:
I would like to see this not being run every time we lint. What you you think about just checking if `shadow` is installed and only then installing it?

Reply:
If we just drop `-u`, we essentially get that behaviour for free. It's just that we might end up with "old" versions of the `shadow` binary. I'm personally OK with that, and expect it would get resolved along with transitioning with go modules.

Reply:
You'r right. Let's just drop `-u` and see if this will works on in the future.

Reply:
```suggestion
	$(GO) get golang.org/x/tools/go/analysis/passes/shadow/cmd/shadow
```